Chaincase.UI/Pages/NewPasswordPage.razor

```diff
@@ -42,12 +46,17 @@
 
     private void SubmitPassword(MouseEventArgs _)
     {
-        ViewModel.SetPassword(Password);
+        try
+        {
+            ViewModel.SetPassword(Password);
+            Global.SetDefaultWallet();
```

Comment:
The view should know no logic beyond the presentation layer (e.g. when a password is submitted). This logic belongs in a ViewModel.

Reply:
Fixed